modules/perception/cpp/farm_ng/perception/frame_grabber_intel.cpp

```
@@ -123,6 +123,7 @@ class FrameGrabberIntel : public FrameGrabber {
       std::stringstream ss;
       ss << "RealSense error calling " << e.get_failed_function() << "("
          << e.get_failed_args() << "):\n    " << e.what();
+      LOG(ERROR) << ss.str();
```

Where a runtime_error was throw, I am getting an error message that looks like the following. It doesn't provide sufficient debugging information. 
```
*** Aborted at 1606030777 (unix time) try "date -d @1606030777" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGABRT (@0x3db8) received by PID 15800 (TID 0x7f86f53450) from PID 15800; stack trace: ***
    @       0x7f8a2856c0 ([vdso]+0x6bf)
    @       0x7f8930e4f8 raise
Aborted (core dumped)
```

This error happened when I didn't properly configure the camera config file to have the properly serial number.

---

Thanks, yeah looks like the error handing scheme globally needs some work.  Not sure what's causing the SIGABRT.

For the framegrabber in particular, some verbosity, such as listing the available cameras might be worthwhile.

---

I googled around that error and tested a tiny bit. It is related to using glog. It hides the actual error somehow. I didn't really look into it.